quipucords/scanner/network/inspect.py

```diff
@@ -110,8 +110,8 @@ def run(self, manager_interrupt):
 
         self.connect_scan_task = self.scan_task.prerequisites.first()
         if self.connect_scan_task.status != ScanTask.COMPLETED:
-            error_message = 'Prerequisites scan task with id %d failed.' %\
-                self.connect_scan_task.id
+            error_message = 'Prerequisites scan task %d failed.' %\
```

Comment:
We should consider adding this error message to our messages file since it is used in 3 different files.

Reply:
Great suggestion. Let's do this as part of https://github.com/quipucords/quipucords/issues/528